app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalArticleController.php

```diff
@@ -47,6 +48,12 @@ class KnowledgeManagementPortalArticleController extends Controller
 {
     public function show(KnowledgeBaseCategory $category, KnowledgeBaseItem $article): JsonResponse
     {
+        $portalViewCountFlag = Feature::active('portal_view_count');
+
+        if ($portalViewCountFlag) {
+            $article->update(['portal_view_count' => $article->portal_view_count + 1]);
```

Comment:
We should use `increment` instead here: https://laravel.com/docs/11.x/queries#increment-and-decrement

`increment` is more performant and is an atomic operation that will prevent issues if multiple people are loading the page simultaneously.

Reply:
Done